cyences_app_for_splunk/default/data/ui/views/cs_mssql.xml

```diff
@@ -0,0 +1,60 @@
+<form version="1.1" theme="light">
+  <label>MSSQL</label>
+  <fieldset submitButton="false">
+    <input type="time" token="tkn_time" searchWhenChanged="true">
+      <label>Time Range</label>
+      <default>
+        <earliest>-24h@h</earliest>
+        <latest>now</latest>
+      </default>
+    </input>
+    <input type="text" token="tkn_host" searchWhenChanged="true">
+      <label>Host</label>
+      <prefix>host="</prefix>
+      <suffix>"</suffix>
+      <default>*</default>
+      <initialValue>*</initialValue>
+    </input>
+    <input type="text" token="tkn_user" searchWhenChanged="true">
+      <label>User</label>
+      <default>*</default>
+      <prefix>database_principal_name="</prefix>
+      <suffix>"</suffix>
+      <initialValue>*</initialValue>
+    </input>
+    <input type="text" token="tkn_action_name" searchWhenChanged="true">
+      <label>Action Name</label>
+      <default>*</default>
+      <prefix>action_name="</prefix>
+      <suffix>"</suffix>
+      <initialValue>*</initialValue>
+    </input>
+  </fieldset>
```

Comment:
Need to add panel for data is present or not for both dashboards.

Reply:
I think Data Present panel is not useful inside dashboard because we don't store host details like which host are reporting. So it kinda always show that host is active.

@VatsalJagani @hardikhdholariya what do you think?

Reply:
@hardikhdholariya  - I would suggest with the help of product_list.py file, can we add dynamically to all dashboards, with minimal hard-coded queries? Not urgent but lets create a separate ticket.

Reply:
Ok, I'll create the ticket.